JDI.Light/JDI.Light/Elements/Composite/WebPage.cs

```diff
@@ -88,6 +88,7 @@ public bool IsOpened
                         return url.Matches(UrlTemplate);
                     case CheckPageType.Contains:
                         return url.Contains(string.IsNullOrEmpty(UrlTemplate) ? Url : UrlTemplate);
+                    default: break;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty 'default' clause.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5599701)